core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -146,6 +146,10 @@ otherCommand
     | beginTxn
     | declareCommand
     | let
+    | scriptingCommand
+    ;
+
+scriptingCommand: BANG set | STRING
```

Comment:
this rule is incorrect. it has to be something like `BANG STRING`. The name `scriptingCommand` is also incorrect, it has to be  snowsqlCliCommand - https://docs.snowflake.com/en/user-guide/snowsql-use. Rename it in python code as well.

Add `SnowsqlCliCommand` into `extensions.scala` and code generate it as the UnresolvedCommand. Add 100% unit test coverage.

Also add `&` to the lexer and create SnowsqlCliVariable. Make sure that it's covered in `SnowflakeToDatabricksTranspilerTest`.

Reply:
@nfx  just saw this comment, you want the ir to defined for `snowsqlclicommand` or to resolved as `unresolvedcommand` ?

Reply:
unresolved is probably better